ERCS/erc-8888.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 8888
```

Comment:
```suggestion
eip: 7602
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
ok